Resources/config/orm.xml

```diff
@@ -145,7 +145,7 @@
             <tag name="doctrine.event_subscriber" />
         </service>
         <service id="doctrine.orm.listeners.pdo_session_handler_schema_subscriber" class="Symfony\Bridge\Doctrine\SchemaListener\PdoSessionHandlerSchemaSubscriber">
-            <argument type="collection" /> <!-- PdoSessionHandler instances -->
+            <argument type="tagged" tag="session.handler_abstract" /> <!-- AbstractSessionHandler instances -->
```

Comment:
`<argument type="service" id="session.handler" />`? but we should also ensure that the service exists, and remove the listener if not
maybe we can do this in DependencyInjection/Compiler/CacheSchemaSubscriberPass.php?

Reply:
I don't think my approach is right in the CacheSchemaSubscriberPass. when I remove the listener if the service doesn't exist I end up with 'doctrine.dbal.default_connection.event_manager' which always has a dependency on the listener.
Thanks for your feedback 🙏